buf/registry/module/v1beta1/label_service.proto

```diff
@@ -98,9 +98,8 @@ message ListLabelsRequest {
   ResourceRef resource_ref = 3 [(buf.validate.field).required = true];
   // The order to return the Labels.
   //
-  // If not specified, defaults to ORDER_CREATE_TIME_ASC.
+  // If not specified, defaults to ORDER_CREATE_TIME_DESC.
   //
-  // TODO: Do we want ORDER_CREATE_TIME_ASC to be the default?
   // TODO: We are purposefully not making the default the zero enum value, however
```

Comment:
Can you explain this decision? If we are going to treat unspecified as a certain enum value anyway (instead of erroring), it seems like we should rename the unspecified value. Is your thinking that if we have an unspecified value as the default that means we are allowed to change the default ordering?

Reply:
A guess: we may wish to change the default but can't change the enum numbering if the API is stable. So clients relying on sorting specify the ordering otherwise unspecified is documented.

Reply:
Just as a general principle, I never use zero values of enums as having meaning - sometimes it's overboard